packages/types/src/interfaces/contractsAbi/definitions.ts

```diff
@@ -170,6 +184,16 @@ const spec = {
     returnType: 'Option<ContractTypeSpec>',
     docs: 'Vec<Text>'
   },
+  ContractMessageSpecV3: {
+    label: 'Text',
+    selector: 'ContractSelector',
+    mutates: 'bool',
+    payable: 'bool',
+    default: 'bool',
```

Comment:
This is not correct - we cannot introduce fields retrospectively to existing versions. In v3 “default” was not available, it is o lay there from v5+

Reply:
Ok, thank you. I wasn't certain about versioning here.
Would something like this be ok ?
```
  ContractMessageSpecV3: 'ContractMessageSpecV2',
  ContractMessageSpecV4: 'ContractMessageSpecV2',
  ContractMessageSpecV5: {
    label: 'Text',
    selector: 'ContractSelector',
    mutates: 'bool',
    payable: 'bool',
    default: 'bool',
    args: 'Vec<ContractMessageParamSpecV2>',
    returnType: 'Option<ContractTypeSpec>',
    docs: 'Vec<Text>'
  },```

Reply:
@jacogr what do you think about my proposition? If it's still not correct, I kindly ask you to explicitly point how to name these types 🙃